README.md

```diff
@@ -0,0 +1,46 @@
+# nodebb-plugin-openedx-discussion (A NodeBB plugin for Openedx)
+
+By integrating this plugin we can use all the functionalities of NodeBB Discussion Forum in Openedx.
+
+## Getting Started
+
+### Prerequisites
+
+You must have an Openedx instance and a NodeBB instance.
+
+### Enabling the plugin
+
+To enable this plugin we need to configure both of our Openedx and NodeBB instances according to following steps:
+
+1. Install [openedx-nodebb-discussion](https://github.com/arbisoft/openedx-nodebb-discussion) in  app in your Openedx instance using its [README](https://github.com/arbisoft/openedx-nodebb-discussion/blob/master/README.md). 
+2. Integrate [nodebb-plugin-openedx-discussion](https://github.com/arbisoft/nodebb-plugin-openedx-discussion) plugin in your NodeBB instance, using the Installation Guide given bellow.
+
+
+### Installation Guide
+
+TO install `nodebb-plugin-openedx-discussion` follow the following steps:
+
+1. Run your NodeBB instance, in the admin panel go to `Extend->Plugins`, in `Plugin Search` search for `nodebb-plugin-openedx-discussion` then install and activate this plugin and restart NodeBB.
+2. Again go to admin panel, in `Plugins` tab select `Openedx Discussion` option.
+3. Enter `JWT cookie name` and `Secret`, make sure that these settings match with the settings of your [openedx-nodebb-discussion](https://github.com/arbisoft/openedx-nodebb-discussion) app.
+
+The Plugin is working now, you can check it by creating a course in you openedx instance with nodebb enabled and selecting the `NodeBB Discussion` tab where a NodeBB iframe will be shown to you. 
```

Comment:
Creating a course is a daunting task in Open edX. There should be a simpler way of verifying that the plugin is working correctly.

Reply:
We can verify that plugin is working by opening `/embed?category_id=<id>` in the browser. But the user should be logged into openEdx first.